intellij-plugin-structure/tests/src/test/kotlin/com/jetbrains/plugin/structure/mocks/Extensions.kt

```diff
@@ -0,0 +1,22 @@
+package com.jetbrains.plugin.structure.mocks
+
+import com.jetbrains.plugin.structure.intellij.plugin.IdePluginImpl
+import org.jdom2.Element
+
+class MockExtension(private val fullyQualifiedName: String, private val elements: List<Element>) {
+  fun apply(plugin: IdePluginImpl) {
+    plugin.extensions[fullyQualifiedName] = elements.toMutableList()
+  }
+
+  companion object {
+    fun from(extensionLocalName: String, vararg attributes: Pair<String, String>): MockExtension {
```

Comment:
we should _always_ work only with FQN of EPs everywhere
e.g. there are some `org.jetbrains` EPs that ship with our plugins etc.

Reply:
Migrated to FQN.